src/types/cls.ts

```diff
@@ -81,14 +81,14 @@ export interface CLSMetricWithAttribution extends CLSMetric {
 /**
  * A CLS-specific version of the ReportCallback function.
  */
-export interface CLSReportCallback extends ReportCallback {
+export interface CLSReportCallback extends Omit<ReportCallback, 'metric'> {
```

Comment:
Is anything gained by doing it this way rather than just doing this (without the `extends`)?

```suggestion
export interface CLSReportCallback {
```

Reply:
I swear I tried that before and [couldn't even get it to build](https://github.com/GoogleChrome/web-vitals/issues/351#issuecomment-1588266261). But it works now, so must have been something else wrong! Will remove this as suggested.